SU-SWS/stanford_components
# EARTH-216: Accessibility Improvements From AMP Report

# READY FOR REVIEW/

# Summary
- Adds title and role to SVG Elements
- Adds role=navigation to collapsible menu 
- Adds LI elements to the UL element in the collapsible menu (was just rendering the a tag before)
- Added many more aria-labels to correctly announce the right thing when tab navigating

# Needed By (Date)
- Before the end of the sprint

# Urgency
- Low

# Steps to Test

1. Check out this branch
2. Clear caches
3. Review /pattern page source for markup changes
4. Enable voice over and tab through the page evaluating the spoken feedback.

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
